platform.md

```diff
@@ -311,14 +309,13 @@ Usage:
 | `<tag>...`        |                       |                          | Additional tag to apply to exported image
 | `<uid>`           | `CNB_USER_ID`         |                          | UID of the stack `User`
 
--`<image>` MUST be a valid image reference
 - **If** the platform provides one or more `<tag>` inputs, each `<tag>` MUST be a valid image reference.
 - **If** `<daemon>` is `false`, `<previous-image>`, if provided,  MUST be a valid image reference
 - **If** `<daemon>` is `true`, `<previous-image>`, if provided, MUST be either a valid image reference or an imageID
 - **If** `<run-image>` is not provided by the platform the lifecycle MUST [resolve](#run-image-resolution) the run image from the contents of `stack` or fail if `stack` does not contain a valid run image.
 - The lifecycle MUST accept valid references to non-existent images without error.
 - The lifecycle MUST fail if the stack ID of the `<run-image>` does not match `<stack-id>`
-- The lifecycle MUST ensure registry write access to `<image>` and any provided `<tag>`s.
+- The lifecycle MUST ensure registry write access to `<previous-image>` and any provided `<tag>`s.
```

Comment:
This doesn't make sense to me. I think we need to support the case where the export destination tag for the app image and the previous image are different. In that case we only need write access to `<image>` and `<tag>`s. The way I understand this working (going off what `creator` does currently)
- export destinations -> `<image>` + `<tag>`s
- previous image -> `<previous-image>` but default to `<image>` if `<previous-image>` is unset